lib/mayaHydra/flowViewportAPIExamples/flowViewportAPIMayaLocator/flowViewportAPIMayaLocator.cpp

```diff
@@ -271,7 +271,7 @@ namespace
 }//end of anonymous namespace
 
 //Initialization of static members
-MTypeId FlowViewportAPIMayaLocator::id( 0x00080100 );
+MTypeId FlowViewportAPIMayaLocator::id( 0x00080101 );
```

Comment:
As per
https://git.autodesk.com/maya3d/maya/pull/27570
this is no longer correct, should be in the range 0x58000086 - 0x5800008F